src/main/Application.java

```diff
@@ -8,10 +9,11 @@
 import java.nio.file.Path;
 
 public class Application extends JFrame {
-
-    public static void appMain() {
+    private static Database _db = null;
+    public static void appMain(Database db) {
```

Comment:
Application is instance of this entire software so it should be responsible for menaging of database insatance. Just create getter for it and insatntiate it in the constructor. I can still be static then getter would look like the "getInstance" one.